src/auth/tests/mod.rs

```diff
@@ -45,20 +45,21 @@ fn test_permission_checker() {
     let checker: PermissionCheckerRef = Arc::new(DummyPermissionChecker);
 
     let grpc_result = checker.check_permission(
-        None,
+        auth::userinfo_by_name(None),
```

Comment:
@shuiyisong In this patch, I changed empty user from `None::Option<UserInfoRef>` to `auth::userinfo_by_name(None)`. Just to confirm it won't break some of our assumptions.